src/gis.h

```diff
@@ -0,0 +1,110 @@
+#ifndef CYCLUS_SRC_GIS_H_
+#define CYCLUS_SRC_GIS_H_
+
+#include <math.h>
+#include <stdio.h>
+#include <sstream>
+#include <string>
+
+#include <boost/shared_ptr.hpp>
```

Comment:
(it can be useful to pass smart shared pointers using this library, but I don't see that functionality currently being used in this code, so including the header file may not be necessary. . .)